### PR TITLE
Import Cluster Refactor

### PR DESCRIPTION
**Pull request by antongisli**
## Describe the Change

This PR builds upon original word edits introduced by @antongisli. The entire import cluster section has been refactored to conform to the style guide and how-to layout guides.

## Review Changes

- I[ndex Page Refactor](https://deploy-preview-1356--docs-spectrocloud.netlify.app/clusters/imported-clusters)
- [Import a Cluster Guide](https://deploy-preview-1356--docs-spectrocloud.netlify.app/clusters/imported-clusters/cluster-import)
- [Attach an Add-on Profile](https://deploy-preview-1356--docs-spectrocloud.netlify.app/clusters/imported-clusters/attach-add-on-profile)
- [Migrate to Full Permissions](https://deploy-preview-1356--docs-spectrocloud.netlify.app/clusters/imported-clusters/migrate-full-permissions)

An additional entry to the Cluster Profile section was added to address the lack of a how-to for creating an add-on cluster profile. 
- [Create an Add-on Profile](https://deploy-preview-1356--docs-spectrocloud.netlify.app/cluster-profiles/create-add-on-profile)

💻 [Preview URL](https://deploy-preview-1356--docs-spectrocloud.netlify.app/clusters/imported-clusters)

### CODE REVIEW

**Comment by karl-cardenas-coding** [comment]
Tagging @ritawatson for a review as I authored the refactored content.

File: content/docs/04-clusters/05-imported-clusters/10-cluster-import.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+   configmap/version-info-kbk5hk992f created
+   secret/spectro-image-pull-secret created
+   priorityclass.scheduling.k8s.io/spectro-cluster-critical created
+   deployment.apps/cluster-management-agent-lite created
+   configmap/cluster-info created
+   configmap/hubble-info created
+   secret/hubble-secrets created
+   ```
+
+9. Once the Palette agent completes the initialization, the side view drawer on the right will disappear, and your cluster will transition to a status of **Running** after a few moments.
```
Comment (ritawatson) [inline content/docs/04-clusters/05-imported-clusters/10-cluster-import.md:211]: When the Palette agent completes initializing, the cluster import procedures at right will disappear, and your cluster will transition to **Running** status within a few minutes.

File: content/docs/04-clusters/05-imported-clusters/10-cluster-import.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+   <br />
+
+   ![A view of an imported cluster's details page](/clusters_imported-clusters_read-only.png)
+
+   <br />
+
+
+You now have a cluster imported into Palette in read-only mode. Keep in mind that a cluster imported in read-only mode has limited capabilities. You can migrate to Full-Permissions at any time by clicking on the **Migrate To Full Permissions**.
```
Comment (ritawatson) [inline content/docs/04-clusters/05-imported-clusters/10-cluster-import.md:220]:
```suggestion
You now have imported a cluster into Palette in read-only mode. Keep in mind that a cluster imported in read-only mode has limited capabilities. You can migrate to full permissions anytime by clicking **Migrate To Full Permissions**.
```

File: content/docs/04-clusters/05-imported-clusters/30-migrate-full-permissions.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
+---
+title: "Migrate to Full Permissions"
+metaTitle: "Migrate to Full Permissions"
+metaDescription: "Learn how to migrate an imported cluster in read-only mode to full-permissions mode."
```
Comment (ritawatson) [inline content/docs/04-clusters/05-imported-clusters/30-migrate-full-permissions.md:4]:
```suggestion
metaDescription: "Learn how to migrate an imported cluster from read-only mode to full-permissions mode."
```

File: content/docs/04-clusters/05-imported-clusters/30-migrate-full-permissions.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+4. From the cluster details page, click on **Migrate To Full Permissions** to start the migration process. A prompt will ask you to confirm your decision. Select **OK**.
+
+  <br />
+
+  ![The cluster details view with an arrow pointing to the migrate button](/clusters_imported-clusters_migrate-full-permissions_cluster-details-page.png)
+
+  <br />
+
+5. A side view drawer will slide out from the right-hand side of the cluster details page. Copy the command displayed to your clipboard.
```
Comment (ritawatson) [inline content/docs/04-clusters/05-imported-clusters/30-migrate-full-permissions.md:43]:
```suggestion
5. A side view drawer will slide out from the right side of the cluster details page. Copy the displayed command to your clipboard.
```

File: content/docs/04-clusters/05-imported-clusters/30-migrate-full-permissions.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+
+
+6. Open up a terminal on your local workstation and validate you are in the correct Kubernete context. You can use the following command to verify the Kubernetes context. If you find yourself in an incorrect Kubernetes context, switch to the proper context so you are interacting with the imported cluster when using kubectl.
```
Comment (ritawatson) [inline content/docs/04-clusters/05-imported-clusters/30-migrate-full-permissions.md:47]:
```suggestion
6. Open a terminal on your local workstation and validate you are in the correct Kubernete context. You can use the following command to verify the Kubernetes context. If you find yourself in an incorrect Kubernetes context, switch to the proper context so you are interacting with the imported cluster when using kubectl.
```

**Review by ritawatson — APPROVED** [review]
Some comments. I suggest using the term "Attach" (_Attach an Add-on Profile_ instead of _Add an Add-on Profile_) because using two terms confusing.

**Comment by ritawatson** [comment]
@karl-cardenas-coding I saw the following in the doc PR. However, in the Palette VerteX UI, the Kubernetes logo next to "Generic" confuses me when we talk about a cloud environment - I think about Kubernetes as a layer in the profile. Is there such a thing as a generic Kubernetes cloud environment? And is it self-managed?
  
CleanShot 2023-06-26 at 12 34 54

File: content/docs/04-clusters/05-imported-clusters/10-cluster-import.md [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+
+2. Navigate to the left **Main Menu** and select **Clusters**.
+
+
+3. Click on **Add New Cluster** and select **Import Cluster** in the pop-up box.
+
+
+4. Fill out the required information and make the proper selections:
+   * Cluster Name - The name of the cluster you want to import.
+   * Cloud Type - Select the infrastructure environment your cluster resides in. Select **Generic** if the environment list doesn't contain your specific environment but be aware of the limitations with generic clusters.
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/05-imported-clusters/10-cluster-import.md:54]: @ritawatson it's mentioned in the intro.

File: content/docs/04-clusters/05-imported-clusters/10-cluster-import.md [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+   helm repo add bitnami https://charts.bitnami.com/bitnami && \
+   helm install my-release bitnami/metrics-server
+   ```
+
+8. To install the Palette agent, issue the command displayed in the cluster details page **Install the agent** section against the Kubernetes cluster you want to import. The command is customized for your cluster as it contains the assigned cluster ID. Below is an example output of the install command.
+
+   <br />
+
+   ```shell hideClipboard
+   kubectl apply -f https://api.spectrocloud.com/v1/spectroclusters/6491d4a94c39ad82d3cc30ae/import/manifest
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/05-imported-clusters/10-cluster-import.md:86]:
```suggestion
   kubectl apply --file https://api.spectrocloud.com/v1/spectroclusters/6491d4a94c39ad82d3cc30ae/import/manifest
```

File: content/docs/04-clusters/05-imported-clusters/10-cluster-import.md [hunk 8]
```diff
@@ -0,0 +1,245 @@
...
+   ```
+
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/05-imported-clusters/10-cluster-import.md:88]: Nice cath 😄

File: content/docs/04-clusters/05-imported-clusters/10-cluster-import.md [hunk 9]
```diff
@@ -0,0 +1,245 @@
...
+   helm repo add bitnami https://charts.bitnami.com/bitnami && \
+   helm install my-release bitnami/metrics-server
+
+
+8. To install the Palette agent, issue the command displayed in the cluster details page  **Install the** read-only agent** section against the Kubernetes cluster you want to import. The command is customized for your cluster as it contains the assigned cluster ID. Below is an example output of the install command.
+
+   <br />
+
+   ```shell hideClipboard
+   kubectl apply -f https://api.spectrocloud.com/v1/spectroclusters/6491d4a94c39ad82d3cc30ae/import/manifest
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/05-imported-clusters/10-cluster-import.md:191]:
```suggestion
   kubectl apply --file https://api.spectrocloud.com/v1/spectroclusters/6491d4a94c39ad82d3cc30ae/import/manifest
```

File: content/docs/03-cluster-profiles/01.5-create-add-on-profile.md [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+Use the following steps to learn how to create an add-on cluster profile.
+
+# Create an Add-on Profile
+
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+
+2. Navigate to the left **Main Menu** and select **Profiles**.
+
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply content/docs/03-cluster-profiles/01.5-create-add-on-profile.md:33]: Thank you 😄

**Pull request merged**